cli/demo_teeracle.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# setup:
+# run all on localhost:
+#   integritee-node purge-chain --dev
```

Comment:
I agree - especially since you have script code at the bottom that is `only for CI testing`. Then I'd also suggest to clearly distinguish between demo and CI tests. If there is much script code duplication, you can probably extract the shared script code and use it from both the demo and CI script?

Reply:
Maybe I should change the name of the demo to demo_teeracle_whitelist.sh. The script is simple, it was designed as an integration test (IT) but it can be used as a small demo as well.

The purpose is to show/verify that an enclave can update the exchange rate only when 
- it is a registered enclave  
- and that the code used is reliable -> the enclave is in the teeracle whitelist.

Even for the demo, we need to ensure that the whitelist is empty at the start. So it can be seen that the mere registration of the mrenclave is not sufficient.

A more complex demo requires a different design.

Reply:
I see. Could you add this as a documentation comment for this script? This would make it a lot clearer.

Reply:
Yes. And I will improve the log messages.